apps/playground/src/app/modules/components/footer-module/footer-module.component.html

```diff
@@ -0,0 +1,188 @@
+      institution, in charge of the web application.
+    </play-input-property-definition>
+    <play-input-property-definition
+      name="operatorLogoAlt"
+      required="Required if operatorLogoSrc has a value"
+      type="It takes a string.">
+      The alternative text for the logo of the operator. Necessary for
+      accessibility: it MUST have at least the text visible on the logo
+      illustration.
+    </play-input-property-definition>
+    <play-input-property-definition
+      name="appName"
+      type="It takes a string.">
+      The name of your web application.
+    </play-input-property-definition>
+    <play-input-property-definition
+      name="description"
+      type="It takes a string.">
+      A short description of what your application does or useful information.
+    </play-input-property-definition>
+    <play-input-property-definition
+      name="link"
+      default="'/'"
+      type="It takes a string.">
+      The path, corresponding to the Angular <code>Route</code> that you want
+      the user to be redirected to when they click on the institution,
+      operator's logo or your app's name.
+    </play-input-property-definition>
+    <play-input-property-definition
+      name="categoriesLinks"
+      type="FooterCategoryLinks[]">
+      The value must be an array of objects satisfying the <code>FooterCategoryLinks</code> interface : each object must have
+      a <code>title</code> property and a <code>children</code> property, which is an array of objects satisfying the <code>FooterLink</code>
+      interface with a <code>label</code> property, a  <code>href</code> property, and an optional <code>isExternal</code> property.
+    </play-input-property-definition>
+    <play-input-property-definition
+      name="middleLinks"
+      type="FooterLink[]">
+      The value must be an array of objects satisfying the  <code>FooterLink</code>
+      interface: each object must have a <code>label</code> property, a <code>href</code> property, and an optional <code>isExternal</code> property.
+    </play-input-property-definition>
+    <play-input-property-definition
+      name="brands"
+      type="FooterBrand[]">
+      The value must be an array of objects satisfying the  <code>FooterBrand</code>
+      interface: each object must have an <code>image</code> property (containing the path), a <code>href</code> property, and an <code>alt</code> property.
+      <br>
+      The first element of the array is the main brand, it will be displayed on the left, the others will be displayed on the right.
+    </play-input-property-definition>
+    <play-input-property-definition
+      name="bottomLinks"
+      type="FooterLink[]">
+      The value must be an array of objects satisfying the  <code>FooterLink</code>
+      interface: each object must have a <code>label</code> property, a <code>href</code> property, and an optional <code>isExternal</code> property.
```

Comment:
`... a <code>label</code> property, an
      <code>href</code> property ( both have string values ), and an optional
      <code>isExternal</code> boolean property.`